pkg/execution/state/redis_state/redis_state.go

```diff
@@ -30,8 +30,9 @@ var embedded embed.FS
 
 var (
 	// scripts stores all embedded lua scripts on initialization
-	scripts = map[string]*rueidis.Lua{}
-	include = regexp.MustCompile(`-- \$include\(([\w.]+)\)`)
+	scripts          = map[string]*rueidis.Lua{}
+	retriableScripts = map[string]*RetriableLua{}
```

Comment:
In addition to our regular scripts, we also prepare a map of retriable scripts (i.e. scripts that can be called with a retriable client).